com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl

```diff
@@ -316,6 +321,17 @@ float2 ClampAndScaleUVForBilinear(float2 UV)
     return ClampAndScaleUV(UV, _ScreenSize.zw, 0.5f);
 }
 
+// This is assuming full screen buffer and .
+float2 ClampAndScaleUVForBilinearPostProcessTexture(float2 UV)
+{
+    return ClampAndScaleUV(UV, _ScreenSize.zw, 0.5f, _RTHandlePostProcessScale.xy);
+}
+
+float2 PostProcessTexelSize()
```

Comment:
I think this doesn't contain what the functions says and also shouldn't (should be previous frame scale to be consistent with rest)

Reply:
Probably _PostProcessScreenSize.zw but I don't think we necessarily need this function as we already assume the texel size is in  _ScreenSize.zw

Reply:
FIxed